cluster/kmeans/model.go

```diff
@@ -0,0 +1,15 @@
+package kmeans
+
+import (
+	"github.com/nextmv-io/sdk/measure"
+)
+
+// Model is a model of a k-means clustering problem.
+type Model interface {
+	// ClusterModels returns the cluster models. The cluster models
+	// define the constraints for the clusters.
+	ClusterModels() []ClusterModel
+
```

Comment:
i would like if this could be called `Cluster` rather than `ClusterModels`.  I found it a bit odd to call .ClusterModels() to access the individual clusters

Reply:
actually - i see why you did it this way.